### PR TITLE
Feature/register myself freelancer

### PR DESCRIPTION
**Pull request by larabr**
- Buttons to "Add myself" (only if logged/no claim in progress/not already freelancer) or "Add someone else" as freelancer
- "Add myself" form linked to claim request (requires document submission)

### CODE REVIEW

File: public/components/freelancer-create.js [hunk 1]
```diff
@@ -14,44 +14,72 @@
...
     });
   }
 
+  claim(createdFreelancerId) {
+    let files = document.getElementById('freelancer-claim-form-files').files;
+    if (files.length === 0) {
+      let message = document.getElementById('freelancer-claim-form-message');
+      message.innerHTML = 'No file was given';
+    } else {
+      ajaxRequest('POST', '/claim/new', { ajax : true }, { freelancerId : createdFreelancerId }, function(claim) {
```
Comment (zuccha) [inline public/components/freelancer-create.js]: Possible confusion of `claim` parameter with `claim` function?

File: public/components/freelancer-create.js [hunk 2]
```diff
@@ -115,10 +143,138 @@ class CreationForm extends React.Component {
...
 }
+
+/**
+ * Sends AJAX request and render the correct creation form
+ * @param  {Boolean} isMyself `true` if it should render the "Add myself as freelancer" form
+ */
+function doAjaxAndRenderForm(isMyself) {
+  ajaxRequest("GET",
+    "/category",
+    { ajax : true }, {},
```
Comment (zuccha) [inline public/components/freelancer-create.js]: Even if empty object, go to newline for consistency.

**Review by zuccha — APPROVED** [review]
Seems good

File: public/components/freelancer-create.js (hunk 1 above)
Comment (larabr) [reply public/components/freelancer-create.js]: this is the same you have in the freelancer profile form

**Pull request merged**